apps/indexer/lib/indexer/fetcher/internal_transaction.ex

```diff
@@ -22,8 +22,9 @@ defmodule Indexer.Fetcher.InternalTransaction do
 
   @behaviour BufferedTask
 
-  @default_max_batch_size 10
-  @default_max_concurrency 4
+  @default_max_batch_size 1
+  @default_max_concurrency 1
+  @default_wait_time 1000
```

Comment:
I'm unsure if I understand the process correctly so spare me if I say something bizarre 😅. Why are these defaults needed if default values are already set in the config file?

Reply:
abundance of caution I think. I followed the pattern I saw in other fetchers.